pkg/server/container_start.go

```diff
@@ -240,10 +245,15 @@ func (c *criContainerdService) generateContainerSpec(id string, sandboxPid uint3
 		g.AddProcessEnv(e.GetKey(), e.GetValue())
 	}
 
-	addOCIBindMounts(&g, config.GetMounts())
+	securityContext := config.GetLinux().GetSecurityContext()
```

Comment:
I feel like it would be better to add a function `setOCIPrivileged`, use `setOCIPrivileged` if privileged is set, and use other functions if not.

This groups all privileged logic together, which makes it more clear what privilege a privileged container has.